src/com/actions/scene/getallscenedata.ts

```diff
@@ -1,5 +1,5 @@
 import { Vector3Like } from "three";
-import { COMLight, COMModel, COMPov } from "../../types";
+import { COMLight, COMModel, COMPov, COMPrimitive } from "../../types";
```

Comment:
Would you be so kind and add `type` in front of all type imports?

I know that hasn't been there before either, but in fact it should and I just forgot. Thank you!

Reply:
Sure :), done ✅ thanks 🙏